app/services/foreman/plugin.rb

```diff
@@ -67,6 +67,16 @@ def register(id, &block)
         registered_plugins[id] = plugin
       end
 
+
+      def modify_backtrace_cleaner
+        cleaner = Rails.backtrace_cleaner
+        cleaner.remove_silencers!
+        cleaner.add_silencer do |line|
+          line !~ Rails::BacktraceCleaner::APP_DIRS_PATTERN &&
+              !(all.any? { |plugin| line.include?(plugin.name) })
```

Comment:
I think we also have plugin.root available for pointing to the installation dir, which might be more precise.